global.json

```diff
@@ -1,22 +1,22 @@
 {
   "sdk": {
-    "version": "5.0.100-preview.7.20323.2",
+    "version": "5.0.100-preview.8.20362.3",
```

Comment:
Why do we need to require a higher version of the SDK globally installed?

Reply:
I need to verify the specific version, but I added a feature to enable us to override the look up for Crossgen.exe while still using the built-in R2R targets, so I needed to bump the SDK version so we could consume it.